docs/general/foundation/testnet/README.md

```diff
@@ -72,6 +72,8 @@ Feel free to use other IAS proxies besides the one provided here or [run your ow
   * `00000000000000000000000000000000000000000000000072c8215e60d5bca7`
 * Runtime binary version (or [build your own](https://github.com/oasisprotocol/emerald-paratime/tree/v7.1.0-rc1#building)):
   * [7.1.0-rc1](https://github.com/oasisprotocol/emerald-paratime/releases/tag/v7.1.0-rc1)
+* Emerald Web3 Gateway version:
+  * [1.3.1](https://github.com/oasisprotocol/emerald-web3-gateway/releases/tag/v1.3.1)
```

Comment:
It should be [2.0.0-rc1](https://github.com/oasisprotocol/emerald-web3-gateway/releases/tag/v2.0.0-rc1) for testnet.

Reply:
Thanks, fixed!